From 76b588c0613c05287b9a28bb73d73287aebb886a Mon Sep 17 00:00:00 2001
From: Takashi Iwai <tiwai@suse.de>
Date: Wed, 29 Jul 2026 09:45:18 +0200
Subject: [PATCH 08182/15333] ALSA: usb-audio: Make some quirk-string helpers
 local

As snd_usb_quirk_flags_from_name() is used only locally, make it
local.  Also, drop the unused snd_usb_quirk_flag_find_name(), too.

Only a code cleanup, no functional changes.

Signed-off-by: Takashi Iwai <tiwai@suse.de>
Link: https://patch.msgid.link/20260729074523.92761-4-tiwai@suse.de
---
 sound/usb/quirks.c | 10 +---------
 sound/usb/quirks.h |  3 ---
 2 files changed, 1 insertion(+), 12 deletions(-)

diff --git a/sound/usb/quirks.c b/sound/usb/quirks.c
index 77adefe4f9cb..3c1343da45b4 100644
--- a/sound/usb/quirks.c
+++ b/sound/usb/quirks.c
@@ -2630,15 +2630,7 @@ static const char *const snd_usb_audio_quirk_flag_names[] = {
 	NULL
 };
 
-const char *snd_usb_quirk_flag_find_name(unsigned long index)
-{
-	if (index >= ARRAY_SIZE(snd_usb_audio_quirk_flag_names))
-		return NULL;
-
-	return snd_usb_audio_quirk_flag_names[index];
-}
-
-u64 snd_usb_quirk_flags_from_name(const char *name)
+static u64 snd_usb_quirk_flags_from_name(const char *name)
 {
 	int i;
 
diff --git a/sound/usb/quirks.h b/sound/usb/quirks.h
index 54f4cca8d7af..0a723c266bb4 100644
--- a/sound/usb/quirks.h
+++ b/sound/usb/quirks.h
@@ -56,7 +56,4 @@ void snd_usb_init_quirk_flags_table(struct snd_usb_audio *chip);
 void snd_usb_init_quirk_flags_parse_string(struct snd_usb_audio *chip,
 					   const char *str);
 
-const char *snd_usb_quirk_flag_find_name(unsigned long flag);
-u64 snd_usb_quirk_flags_from_name(const char *name);
-
 #endif /* __USBAUDIO_QUIRKS_H */
-- 
2.55.0

